tuna.go

```diff
@@ -745,6 +750,21 @@ func (c *Common) CreateServerConn(force bool) error {
 				return err
 			}
 
+			entryToExitMaxPrice, exitToEntryMaxPrice, err := ParsePrice(c.ServiceInfo.MaxPrice)
```

Comment:
Only need to check these if `minBalance > 0`

Reply:
Yes, modified.